File: app/src/main/java/com/aarushi/restapiretrofitjson/Models/Thumbnail.java
package com.aarushi.restapiretrofitjson.Models;

/**
 * Created by hp on 7/1/2017.
 */

public class Thumbnail {

    String title;
    String thumbnailUrl;
    String url;
    public Thumbnail(String title, String thumbnailURL, String url) {
        this.title = title;
        this.thumbnailUrl = thumbnailURL;
        this.url=url;

    }

    public String getUrl() {
        return url;
    }


    public String getTitle() {
        return title;
    }

    public String getThumbnailUrl() {
        return thumbnailUrl;
    }
}

File: app/src/main/java/com/codingblocks/restapiretrofitjson/models/Thumbnail.kt
package com.codingblocks.restapiretrofitjson.models

/**
 * Created by Harshi on 7/5/2017.
 */
class Thumbnail(title: String, thumbnailURL: String, url: String) {
    var title: String
        internal set
    var thumbnailUrl: String
        internal set
    var url: String
        internal set

    init {
        this.title = title
        this.thumbnailUrl = thumbnailURL
        this.url = url

    }
}